examples/jquery-bootstrap/js/app.js

```diff
@@ -19,9 +19,13 @@ var BootstrapModal = React.createClass({
   componentDidMount: function() {
     // When the component is added, turn it into a modal
     $(this.refs.root).modal({backdrop: 'static', keyboard: false, show: false});
+
+    // Bootstrap's modal class exposes a few events for hooking into modal
+    // functionality. Lets hook into one of them:
+    $(this.refs.root).on('hidden.bs.modal', this.handleHidden);
   },
   componentWillUnmount: function() {
-    $(this.refs.root).off('hidden', this.handleHidden);
+    $(this.refs.root).off('hidden.bs.modal', this.handleHidden);
```

Comment:
Actually, both event names work. The syntax is [how jQuery namespaces events](https://css-tricks.com/namespaced-events-jquery/). So it wasn't working for me initially because there wasn't an event listener setup when the component was mounted.